packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -120,7 +152,7 @@ class LinkData with SearchableDataMixin {
   final List<PlatformOS> os;
   final List<String> scheme;
   final List<DomainError> domainErrors;
-  final bool pathError;
+  List<PathError> pathErrors;
```

Comment:
same question for all of these actually. Are any of these supposed to have unique, non-repeating content? If so, should we use a Set?